defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 firewall: []
 firewall_disable_conflicting_services: false
+transactional_update_reboot_ok: null
```

Comment:
> Would it be better to use transactional_update_ or __firewall_is_transactional as a prefix instead? This way, it will be clear that the variables and tasks are related to transactional update systems rather than storage systems.

ansible-lint requires role variables to be prefixed with role name. It's a good practice to group all role's variables for the sake of clarity. So this should be `firewall_transactional_update_reboot_ok` .